.github/workflows/linux_pip.yml

```diff
@@ -27,6 +26,11 @@ jobs:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0
+      - uses: GuillaumeFavelier/ci-skip@enh/separator
+        with:
+          fail-fast: true
+          commit-filter: '[skip ci];[ci skip];[skip github]'
+          separator: ';'
```

Comment:
So they already support this? Then PR to their README would be good

Reply:
That's my fork, I am testing in here before opening the PR :)

Reply:
Ahh missed that, cool!

Reply:
I opened https://github.com/mstachniuk/ci-skip/pull/4